docs/code-of-conduct.md

```diff
@@ -118,15 +118,13 @@ the community.
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage],
 version 2.0, available at
-https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+<https://www.contributor-covenant.org/version/2/0/code_of_conduct.html>.
```

Comment:
I wasn't aware of this syntax, what is it based on?

Reply:
After the CommonMark Spec a Link:
> contains link text (the visible text), a link destination (the URI that is the link destination), [...]  

A [link destination](https://spec.commonmark.org/0.29/#link-destination) is defined as:

> a sequence of zero or more characters between an opening < and a closing > that contains no line breaks or unescaped < or > characters, or [...]

So in this case, It's a Link without the link text.  
The Markdownlint rule is [MD034](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md034---bare-url-used) and the reason for it:
> Without angle brackets, the URL isn't converted into a link by many markdown parsers.

In our case Github/VScode do convert raw URL to links, but the jeykill parser does not.